docs/user_guide/HOWTO/Manage_TPM.rst

```diff
@@ -2,20 +2,22 @@ How to Manage a TPM Device with SIMP
 ====================================
 
 This document serves as a guide to enable and use TPM devices in SIMP.
-Currently, both :term:`TPM` **1.2** and **2.0** are supported, but only on
-EL7 systems.
+Currently, both :term:`TPM` **1.2** and **2.0** are supported, but TPM 2.0
+features are only available on EL7 systems.
 
 TPM features in SIMP:
 
+  * TPM 1.2 and TPM 2.0 simulators
```

Comment:
seems like this should be clear that SIMP provides signed packages of IBM TPM simulator software

Reply:
This is an overview of features; the section on the simulators provides this detail.